app/src/main/java/com/example/pokedexdesafio/data/model/PokemonDetailResponse.kt

```diff
@@ -0,0 +1,77 @@
+package com.example.pokedexdesafio.data.model
+
+import android.os.Parcelable
+import com.example.pokedexdesafio.domain.model.*
+import kotlinx.parcelize.Parcelize
+
+@Parcelize
+data class PokemonDetailResponse(
```

Comment:
All the parameters of something coming from a service should be nullable

Reply:
still pending

Reply:
solved